data/Store.js

```diff
@@ -400,17 +400,17 @@ export class Store extends HoistBase {
     }
 
     /**
-     * Modify individual Record field values in a local, uncommitted state - i.e. when updating a
-     * Record or Records via an inline grid editor or similar control.
+     * Modify individual StoreRecord field values in a local, uncommitted state - i.e. when updating a
```

Comment:
I do think we need to take another pass -- most of these references to StoreRecord would be better as 'record'